server/server.js

```diff
@@ -183,9 +232,55 @@ app.post('/genotet/upload', upload.single('file'), function(req, res) {
   });
 });
 
-/**
- * GET request handler.
- */
+app.use(cookieParser());
+
+app.post('/genotet/user', function(req, res) {
+  log.serverLog('POST user');
+
+  var query = req.body;
+  var type = query.type;
+  var data;
+  var returnCallback = function(data) {
```

Comment:
serverResponse... as in @djasmine 's... But I think @djasmine moved this to outer scope.